app/hotels/src/allHotels/RenderSearchResults.js

```diff
@@ -91,18 +93,14 @@ export const RenderSearchResults = (props: Props) => {
           duration: transitionDuration,
           useNativeDriver: true,
         }),
-      ]).start();
+      ]).start(() => {
+        setShouldRenderMap(true);
```

Comment:
You also want it to be true here? Is that on purpose?

Reply:
No, I was testing something, and I forgot to flip it back, thanks.